test/system/070-build.bats

```diff
@@ -1097,6 +1097,18 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman build empty context dir" {
+    buildcontextdir=$PODMAN_contextdir/emptydir
```

Comment:
You seem to have done a global search/replace. There is no such thing as `PODMAN_contextdir`. This should be `PODMAN_TMPDIR`, same as line 1103 below.